src/main/kotlin/com/github/avrokotlin/avro4k/AnnotationExtractor.kt

```diff
@@ -25,7 +25,8 @@ class AnnotationExtractor(private val annotations: List<Annotation>) {
       }else{
          annotations.filterIsInstance<AvroAliases>().flatMap { it.value.toList() }
       }
-   fun props(): List<Pair<String, String>> = annotations.filterIsInstance<AvroProp>().map { it.key to it.value }
+   fun props(): List<Pair<String, String>> = annotations.filterIsInstance<AvroProp>().map { it.key to it.value }.filterNot { it.second.isBlank() }
+   fun propsList(): List<Pair<String, List<String>>> = annotations.filterIsInstance<AvroProp>().map { it.key to it.values.toList() }.filterNot { it.second.isEmpty() }
```

Comment:
Little improvements:
- Allow empty list, as it could be a use case
- Don't convert to a List where it's just after used with values.forEach
```suggestion
   fun propsList(): List<Pair<String, Array<String>>> = annotations.filterIsInstance<AvroProp>().map { it.key to it.values }
```